### ISSUE TITLE
fix/audits/buttons missing accessible names

### ISSUE THREAD

**Issue by funkaj** [funkaj/portfolio-react#6]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

https://dequeuniversity.com/rules/axe/3.1/button-name?application=lighthouse
